data/fields/barrier.json

```
@@ -1,5 +1,32 @@
 {
     "key": "barrier",
     "type": "typeCombo",
-    "label": "Type"
+    "label": "Type",
+     "strings": {
```

PS: the indentation does have one too many space character here and the lines below.

```suggestion
    "strings": {
```